dapps/profile.vue/src/components/wallet/wallet.ts

```diff
@@ -50,6 +56,8 @@ export default class WalletComponent extends mixins(EvanComponent) {
    * Bin window resize watcher to handle side panel state and handle send eve events.
    */
   async created() {
+    this.loadBalance();
```

Comment:
```suggestion
    await this.loadBalance();

    this.loading = false;
```

The loadBalance function is asynchronous and should be awaited. Should a loading symbol be shown until balance is loaded? If not, we can simply remove the loading parameter from the class.